### PR TITLE
Update pin to hdf5 1.8.18

### PR DESCRIPTION
**Pull request by qwhelan**
Update ```hdf5``` to 1.8.18 to resolve some CVEs announced back in November.

See https://github.com/conda-forge/hdf5-feedstock/issues/71 for more info.

### CODE REVIEW

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
           'glib': 'glib 2.51.*',  # 2.50.2
           'gmp': 'gmp >=5.0.1,<7', # 6.1.0
           'harfbuzz': 'harfbuzz 1.3.*',  # 0.9.39
-          'hdf5': 'hdf5 1.8.17|1.8.17.*',  # 1.8.17
+          'hdf5': 'hdf5 1.8.18|1.8.18.*',  # 1.8.17
```
Comment (ocefpaf) [inline scripts/pin_the_slow_way.py]: @jjhelmus can you confirm if `defaults` version is still `1.8.17`?

(It is probably safe for `conda-forge` to move to `1.8.18` b/c we do have all the packages that uses `hdf5` packaged in `conda-forge`.)

Comment (qwhelan) [inline scripts/pin_the_slow_way.py]: It is still 1.8.17 - I have a PR to update it here: https://github.com/conda-forge/hdf5-feedstock/pull/68

In my experience, my PRs get ignored even if it's about a remote code execution vuln.

Comment (ocefpaf) [inline scripts/pin_the_slow_way.py]: @qwhelan are you referring to `defaults` or `conda-forge`? I am confused b/c you linked the PR to `conda-forge`, but that was merged and we **are** moving to the latest `hdf5` thanks to you.

Comment (qwhelan) [inline scripts/pin_the_slow_way.py]: @ocefpaf Apologies, pasted the wrong link initially. I was referring to ```defaults```

**Pull request merged**